features.tex

```diff
@@ -2,10 +2,9 @@
 
 % Basic operations (the core)
 \subsection{Basic Operations}
-% Including the assumptions system
```

Comment:
If we move the features section to the beginning of the paper, we should include some description of how to define symbols and create expressions.